Poort8.Ishare.Service/Service/ServiceController.cs

```diff
@@ -133,9 +133,9 @@ public class ServiceController : ControllerBase
     {
         var authorization = Request.Headers.Authorization;
 
-        _logger.LogDebug("Received service PUT request with authorization header: {authorization}", authorization);
+        _logger.LogDebug("Received service PUT request with authorization header: {authorization}", authorization!);
```

Comment:
Change `authorization!` to `authorization ?? "NULL"`, to log when authorization is null?